src/systems/ec/specs/components/select/demo/data--single.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
     {
       value: '6',
       label: 'Italy',
+      isDisabled: true,
```

Comment:
Can we handle `disabled` only as well? If we go for spread of options as described below, passing any attribute key:value pair will be put directly.